pages/common/unrar.md

```diff
@@ -0,0 +1,15 @@
+# unrar
+
+> Extract RAR archives
+
+- extract files with full path
+
+`unrar x {{compressed.rar}}`
+
+- test archive files
```

Comment:
The description should be "test an archive file", to match the specific example below.